Bencodex.Tests.ruleset

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RuleSet
+        Name="Rules for Bencodex.Test"
+        Description="Code analysis rules for Bencodex.Test.csproj."
```

Comment:
The project name should be plural here, right?

Reply:
Yeah, I will fix it.